librz/bin/format/elf/elf.h

```diff
@@ -319,6 +316,14 @@ RZ_OWN RzVector *Elf_(rz_bin_elf_new_sections)(RZ_NONNULL ELFOBJ *bin);
 RZ_OWN char *Elf_(rz_bin_elf_section_type_to_string)(ut64 type);
 bool Elf_(rz_bin_elf_has_sections)(RZ_NONNULL ELFOBJ *bin);
 
+// elf_strtab
+
+RZ_OWN RzBinElfStrtab *Elf_(rz_bin_elf_new_strtab)(RZ_NONNULL ELFOBJ *bin, ut64 offset, ut64 size);
```

Comment:
Better call it `rz_bin_elf_strtab_new()`

Reply:
Should I change rz_bin_elf_new_sections, rz_bin_elf_new_segments, rz_bin_elf_new_dt_dynamic ?

Reply:
Yes I think that's better since pretty much all our constructors use `<name>_new`.